CONTRIBUTE.md

```diff
@@ -0,0 +1,45 @@
+# Darts Contribution Guidelines
+
+## Main Development Principles
+
+* Focus on simplicity and clarity for end users.
```

Comment:
I'd also specify which users we're aiming with darts - beginners, experienced, pr0s

Reply:
I'm not sure it changes much at this stage. A clear API for a newbie is hopefully also clear for a pro and vice-versa (I agree there might be small differences, but I'm a bit reluctant to drive gear too much the development towards a particular subset of users at this stage).